packages/db-react/src/root-loader.tsx

```diff
@@ -17,17 +17,13 @@ export interface RootLoaderData {
 }
 
 export async function rootLoader() {
-  const [networks, settings, wallets] = await Promise.all([
-    getOrFetchQuery(queryClient, optionsNetwork.findMany()),
+  const [accounts, networks, settings, wallets] = await Promise.all([
+    getOrFetchQuery(queryClient, optionsAccount.findMany({})),
+    getOrFetchQuery(queryClient, optionsNetwork.findMany({})),
     getOrFetchQuery(queryClient, optionsSetting.getAll()),
-    getOrFetchQuery(queryClient, optionsWallet.findMany()),
+    getOrFetchQuery(queryClient, optionsWallet.findMany({})),
   ])
 
-  const activeWalletId = settings.find((s) => s.key === 'activeWalletId')?.value
-  const accounts = activeWalletId
-    ? await getOrFetchQuery(queryClient, optionsAccount.findMany({ walletId: activeWalletId }))
-    : []
-
   return {
     accounts,
```

Comment:
Accounts here will now be different from the accounts in the live query here `useAccountsForWalletLive`. 1st render will load all accounts and render after hydration will load only accounts filtered by wallet id. 

I'm fine merging this now if we handle this immediately after.